packages/react-lowcode/src/codegen/generation/generators/detail/grommet-detail-generator.ts

```diff
@@ -20,12 +20,15 @@ export default class GrommetDetailGenerator
   private _context: GenerationContext
   private _entity: Entity
   private _intlFormatter: ReactIntlFormatter
+  private _entityName: string = ""
 
   constructor(generationContext: GenerationContext, entity: Entity) {
     this._helper = new GeneratorHelper(generationContext, this._imports)
     this._context = generationContext
     this._entity = entity
     this._intlFormatter = new ReactIntlFormatter(generationContext, this._imports)
+    if (entity)
+        this._entityName = entity.getName();
```

Comment:
no need to do this since two lines above you allready assigned `this._entity` (let's not have this duplicate unless necessary)